Sources/AppStoreServerLibrary/ChainVerifier.swift

```diff
@@ -98,11 +78,16 @@ class ChainVerifier {
                 }
             }
         }
+
         let verificationResult = await verifyChainWithoutCaching(leaf: leaf, intermediate: intermediate, online: online, validationTime: validationTime)
-        
+
         if online {
-            if case .validCertificate = verificationResult {
-                verifiedPublicKeyCache[CacheKey(leaf: leaf, intermediate: intermediate)] = CacheValue(expirationTime: getDate().addingTimeInterval(TimeInterval(integerLiteral: ChainVerifier.CACHE_TIME_LIMIT)), publicKey: verificationResult)
+            if case .validCertificate(_) = verificationResult {
```

Comment:
Nit unnecessary change, this looks like just reformatting, could we please keep the reformatting a such to a minimum in this PR (if we just want a reformatting PR that would be fine)